api/app/signals/apps/signals/utils/map.py

```diff
@@ -0,0 +1,78 @@
+import io
+import urllib.request
+from math import asinh, pi, radians, tan
+
+from PIL import Image
+
+TILE_SIZE = 256
```

Comment:
Is this standard across tile servers?

Reply:
most server I tested used 256 tiles, so ..

Reply:
I see that this is the "defacto standard" according to this OpenStreetMap Wiki page: https://wiki.openstreetmap.org/wiki/Tiles